assets/src/components/notifications.tsx

```diff
@@ -1,15 +1,109 @@
-import { useState } from "react"
+import React, { useState } from "react"
+import useInterval from "../hooks/useInterval"
 import { useNotifications } from "../hooks/useNotifications"
 import featureIsEnabled from "../laboratoryFeatures"
-import { Notification } from "../realtime.d"
+import { Notification, NotificationReason } from "../realtime.d"
+import { formattedTimeDiff, now } from "../util/dateTime"
+import PropertiesList from "./propertiesList"
 
 export const Notifications = () => {
-  const [, setNotifications] = useState<Notification[]>([])
+  const [currentTime, setCurrentTime] = useState(now())
+  useInterval(() => setCurrentTime(now()), 1000)
+
+  const [notifications, setNotifications] = useState<Notification[]>([])
   const addNotification = (notification: Notification): void => {
     if (featureIsEnabled("notifications")) {
       setNotifications((previous) => [...previous, notification])
     }
   }
+  const removeNotification = (id: number): void => {
+    setNotifications((previous) => previous.filter((n) => n.id !== id))
+  }
   useNotifications(addNotification)
-  return null
+  return (
+    <div className="m-notifications">
+      {notifications.map((notification) => (
+        <NotificationCard
+          key={notification.id}
+          notification={notification}
+          remove={removeNotification}
+          currentTime={currentTime}
+        />
+      ))}
+    </div>
+  )
+}
+
+const NotificationCard = ({
+  notification,
+  remove,
+  currentTime,
+}: {
+  notification: Notification
+  remove: (id: number) => void
+  currentTime: Date
+}) => (
+  <div className="m-notifications__card">
+    <button className="m-notifications__card-info" onClick={() => {}}>
```

Comment:
@skyqrose may as well just remove the `onClick` handler then, unless I'm missing something?